content/en/docs/rules/_index.md

```diff
@@ -251,11 +251,20 @@ Last but not least, it's also possible to just disable a by default enabled rule
   enabled: false
 ```
 
-This is especially useful for rules which do not provide a `consider_*` macro in it's default condition. Again, ensure the falco configuration files are loaded in the right order.
+This is especially useful for rules which do not provide a `consider_*` macro in it's default condition. Again, ensure the Falco configuration files are loaded in the right order.
```

Comment:
How about?
Last but not the least, you can just disable a rule that is enabled by default using a combination of the `append: true` and `enabled: false` rule properties.
This is especially useful for rules which do not provide a `consider_*` macro in the default condition.

Ensure that the custom configuration file loads after the default configuration file. You can configure the right order using multiple `-r` parameters, directly inside the falco configuration file `falco.yaml` through `rules_file`. If you are using the official Helm chart, then configure the order with the `falco.rulesFile` value.

For example to disable the `User mgmt binaries` default rule in `/etc/falco/falco_rules.yaml` define a custom rule in `/etc/falco/rules.d/custom-rules.yaml`:

```yaml
- rule: User mgmt binaries
  append: true
  enabled: false
```


 {{% pageinfo color="warning" %}}
 There appears to be a bug with this feature that we are looking into. If `enabled: false` doesn't work, you can use the following workaround as an alternative:
 ```yaml
 - rule: User mgmt binaries
   condition: and (never_true)
   append: true
 ```
 {{% /pageinfo %}}

Reply:
Thank you! much better